src/main/kotlin/dev/fff/fractalizer/model/FitnessFunction.kt

```diff
@@ -0,0 +1,12 @@
+package dev.fff.fractalizer.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+
+class FitnessFunction(
+        val name: String,
+        @JsonInclude(JsonInclude.Include.NON_NULL)
+        val description: String?,
+        val okay: Boolean,
+        @JsonInclude(JsonInclude.Include.NON_NULL)
+        val children: List<FitnessFunction>?
```

Comment:
map settings toevoegen

Reply:
Will add it when we will actually use it.